build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -103,13 +102,20 @@
     <Copy SourceFiles="@(HackFilesToCopy)"
           DestinationFiles="@(HackFilesToCopy->'$(SdkOutputDirectory)/%(RecursiveDir)%(Filename)%(Extension)')" />
     
-      <!-- Publish DotNet -->
+    <!-- Publish DotNet -->
     <DotNetPublish ToolPath="%(Stage.DotnetDir)"
-                                Output="$(SdkOutputDirectory)"
-                                Configuration="$(Configuration)"
-                                VersionSuffix="$(CommitCount)"
-                                ProjectPath="$(SrcDirectory)/redist/redist.csproj"
-                                MSBuildArgs="/p:GenerateRuntimeConfigurationFiles=true"/>
+                   Output="$(SdkOutputDirectory)"
+                   Configuration="$(Configuration)"
+                   VersionSuffix="$(CommitCount)"
+                   ProjectPath="$(SrcDirectory)/redist/redist.csproj"
+                   MSBuildArgs="/p:GenerateRuntimeConfigurationFiles=true" />
+
+    <!-- Publish DotNet -->
+    <DotNetPublish ToolPath="%(Stage.DotnetDir)"
+                   Output="$(SdkOutputDirectory)/Roslyn"
+                   Configuration="$(Configuration)"
+                   VersionSuffix="$(CommitCount)"
+                   ProjectPath="$(SrcDirectory)/tool_roslyn/tool_roslyn.csproj" />
```

Comment:
Note, did not duplicate `/p:GenerateRuntimeConfigurationFiles=true`, setting the property in tool_csproj instead. This and other changes allow that csproj file to be independently build/publish-able. That greatly improved my dev cycle.